src/app/(app)/movies/[movieId]/page.tsx

```diff
@@ -18,9 +18,8 @@ export default async function SingleMoviePage({ params }: Context) {
 
   return (
     <>
-      <MovieDetailSection movie={movie} />
+      {/* <MovieDetailSection movie={movie} /> */}

```

Comment:
I noticed MovieDetailSection is commented out. Is this for future updates or refactoring purposes?

Would be great to leave this in to showcase the progress so far on the app for tomorrow with the mentors.

Reply:
Oops sorry must've been by accident 🫡